### PR TITLE
groups: use landscape settings

### PR DESCRIPTION
**Pull request by nerveharp**
Dependent on urbit/garden#10.

Implements controls from Landscape to disable:
- avatars
- nicknames
- remote image fetching
- spellcheck

Manifests in the following product locations:
- Chat images
- Ship avatars
- Ship names
- Profile headers
- Notebook headers
- Notebook images
- Gallery items
- Notebook, gallery, profile bio, and chat text inputs
- Group icons
- Group headers

Also introduces a constant for the Landscape desk, since we're using `%garden` now and will eventually use `%landscape`.

fixes tloncorp/landscape-apps#936

### CODE REVIEW

File: ui/src/profiles/ProfileCoverImage.tsx [hunk 1]
```diff
@@ -25,7 +26,8 @@ export default function ProfileCoverImage({
   children,
 }: CoverProps) {
   const contact = useContact(ship);
-  const { cover } = contact || emptyContact;
+  const calm = useCalm();
+  const cover = !calm.disableRemoteContent || contact || emptyContact;
```
Comment (arthyn) [inline ui/src/profiles/ProfileCoverImage.tsx:30]: I think something went wrong here this should have given a type error. Missing destructuring, but also the logic needs to be different I think.

**Review by arthyn — CHANGES_REQUESTED** [review]
Just the one small issue, after that should be good to go

File: ui/src/profiles/ProfileCoverImage.tsx [hunk 2]
```diff
@@ -26,16 +26,20 @@ export default function ProfileCoverImage({
   children,
 }: CoverProps) {
   const contact = useContact(ship);
-  const calm = useCalm();
-  const cover = !calm.disableRemoteContent || contact || emptyContact;
+  const { disableRemoteContent } = useCalm();
+  const cover = contact || emptyContact;
```
Comment (arthyn) [inline ui/src/profiles/ProfileCoverImage.tsx:30]:
```suggestion
  const { cover } = contact || emptyContact;
```

**Review by arthyn — APPROVED** [review]
lgtm with suggestion

**Comment by nerveharp** [comment]
> lgtm with suggestion

Thank you, _that's_ what you meant by destructuring.

**Pull request merged**